src_to_mesh.cpp

```diff
@@ -99,7 +106,11 @@ int main(int argc, char** argv) {
 
 
   std::vector<double> rel_strengths(bin_counts.begin(), bin_counts.end());
-  for (auto& rs : rel_strengths) { rs /= sites_sampled; }
+  for (auto& rs : rel_strengths) { rs /= (sites_sampled-missed_bins); }
+
+  if ((missed_bins) > 10){
```

Comment:
Why not `> 0`?

Reply:
Agreed, this should be 0. I didn't change it after testing